Sources/SwiftUICharts/LabelsView.swift

```diff
@@ -17,14 +19,16 @@ struct LabelsView: View {
     }
 
     var body: some View {
-        HStack(spacing: 0) {
-            ForEach(dataPoints.indexed(), id: \.1.self) { index, bar in
-                if index % self.threshold == 0 {
-                    Text(bar.label)
-                        .multilineTextAlignment(.center)
-                        .foregroundColor(.secondary)
-                        .font(.caption)
-                    Spacer()
+        GeometryReader { geometry in
+            HStack(spacing: 0) {
+                ForEach(dataPoints.indexed(), id: \.1.self) { index, bar in
+                    if index % self.threshold == 0 {
+                        Text(bar.label)
+                            .multilineTextAlignment(.center)
+                            .foregroundColor(axisColor)
+                            .font(.caption)
+                            .frame(width: geometry.size.width / CGFloat(labelCount), alignment: .center)
```

Comment:
This solution breaks the Dynamic Type support.

Reply:
I did not find a more elegant solution to make sure the label is align center with the value in graph in order to have that kind of design mainly due to the fact that graph subview are not overridable. In my develop branch I have worked on a more customisable layout solution.

In case of Dynamic Type Support not sure it break things since if you enlarge font and because it is `multilineTextAlignment` it shall go in multiline with word wrap. Isn't it?

<img width="905" alt="Capture d’écran 2020-11-20 à 09 58 02" src="https://user-images.githubusercontent.com/47981468/99780515-e2d7cb00-2b16-11eb-8dd8-c0580597a213.png">